.github/workflows/playground.yml

```diff
@@ -0,0 +1,42 @@
+name: Build playground
+on:
+  push:
+    tags:
+    - '*'
+jobs:
+  run:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        ocaml-version: ["4.06.1"]
+        node-version: [12.x]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.0.0
+      - name: Cache .opam folder
+        id: cache-opam
+        uses: actions/cache@v1
+        with:
+          path: ~/.opam
+          key: ${{ runner.os }}-opam-${{ matrix.ocaml-version }}-3.5.1-v1
+      - name: Use OCaml ${{ matrix.ocaml-version }}
+        uses: avsm/setup-ocaml@v1.0
+        with:
+          ocaml-version: ${{ matrix.ocaml-version }}
+      - if: steps.cache-opam.outputs.cache-hit != 'true'
+        run: opam install js_of_ocaml.3.5.1
+      - run: git submodule update --init && ./scripts/buildocaml.js
+      - run: ./scripts/ninja.js config && ./scripts/ninja.js build
+      - run: mkdir playground && mkdir playground/stdlib
+      - run: BS_PLAYGROUND=../playground ./scripts/repl.js -prepublish
+      - run: node -e "require('./exports.js'); eval(ocaml.compile('Js.log Sys.ocaml_version').js_code)"
+        working-directory: ./playground
```

Comment:
can this be put in another PR? we evaluated github workflow before, it does not run as fast as circleCI